### PR TITLE
feat(security): security user guide (RS-1997)

### PR DESCRIPTION
**Pull request by mpean**

### CODE REVIEW

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+![OAuth 2.0 logo](images/oauth2.png "OAuth 2.0 logo")
+
+You want to design a secure API? Yes, defining you're API access security is an essential requirement.
```
Comment (constint) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: `defining your`

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+You may want simply define a common security for all your API, or choose a more fine policy with a specific security
```
Comment (constint) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: `more fine` -> `finer`

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- Pass through
+- Custom / API key
+
+Let's se how to do that in Restlet Studio.
+
+# How to design your API security in Restlet Studio
+
+To begin to define your API security you first need to define your security schemes.
+
+Then you can define the global security for all your API by selecting your security schemes you want to apply
```
Comment (constint) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: `selecting the security schemes`

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Then you can define the global security for all your API by selecting your security schemes you want to apply 
+globally. By default, your global security will apply to all your resources and operations in your API.
+
+Finally you can define a different security for each operation or all operations in a resource.
+
+## Define your security schemes and your API global security
+
+To define your security schemes, go to your API's general information and add your schemes.
+Then selected your security schemes you want to apply globally in your API.
```
Comment (constint) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: `Then select the`

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+## Define a specific security on a resource or an operation
+
+To define a specific security for a resource, select your resource and choose your security schemes you
+ want to apply to all your resource's operations. Then, you can select an operation to define its specific security.
+  
+![Resource security panel](images/resource-security.png "Resource security panel")
+
+## That's it, you know how to define your API security!
+Please contact support via the in-app chat widget if you need more informations to define your API security, we're happy
```
Comment (constint) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: `more information`

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md (hunk 1 above)
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: Sentence is a bit awkward to start with.

How about something like: 

"As part of the API design process, you will need to specify which consumers can access your API (authentication) and what they are allowed to do (authorization)"

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+![OAuth 2.0 logo](images/oauth2.png "OAuth 2.0 logo")
+
+You want to design a secure API? Yes, defining you're API access security is an essential requirement.
+
+You may want simply define a common security for all your API, or choose a more fine policy with a specific security 
+for a critical resource?
```
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: remove question mark, or change whole sentence into a question (as sentence starting with "You may want" will rarely end with a question mark).

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md (hunk 2 above)
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: for all your API -> for your entire API

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
+![OAuth 2.0 logo](images/oauth2.png "OAuth 2.0 logo")
+
+You want to design a secure API? Yes, defining you're API access security is an essential requirement.
+
+You may want simply define a common security for all your API, or choose a more fine policy with a specific security 
+for a critical resource? 
+
+You may want to give a free access to your search products operation ?
```
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: You may also want to specify that certain operations to be freely accessible without authentication.

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+No problem, Restlet Studio allows you to define all these cases with the main security types :
```
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: "Restlet Studio can handle all these cases"

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+- Basic authentication
+- Digest authentication
+- OAuth 1.0
+- OAuth 2.0
+- Pass through
+- Custom / API key
+
+Let's se how to do that in Restlet Studio.
```
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: se -> see

**Comment by jmcx** [comment]
Ca pourrait être interessant d'avoir une petite section à la fin pour dire à quoi servent les custom headers and query params. Mais c'est pas bloquant pour la sortie si tu préfères finir une première itération!

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+To define your security schemes, go to your API's general information and add your schemes.
+Then selected your security schemes you want to apply globally in your API.
+
+![Global security panel](images/global-security.png "Global security panel")
+
+
+## Define a specific security on a resource or an operation
+
+To define a specific security for a resource, select your resource and choose your security schemes you
+ want to apply to all your resource's operations. Then, you can select an operation to define its specific security.
```
Comment (jmcx) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: "Then, you can select" -> on a l'impression qu'on doit enchaîner les deux. 

Peut-être plutôt, sur une nouvelle ligne -> "Alternatively, you can select an operation.... "

File: modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md (hunk 6 above)
Comment (mpean) [inline modules/com.restlet/tech-doc/studio/guide/02_editor/05_security.md]: Ok i remove the question mark

**Pull request closed without merging**